### ISSUE TITLE
php:8.0.11-apache-buster - please update Apache version

### ISSUE THREAD

**Issue by xamadev** [docker-library/php#1214]
There a severe security issues with Apache 2.4.38 (https://httpd.apache.org/security/vulnerabilities_24.html) - please update the image with newest Apache version. Thanks!

**Comment by tianon** [comment]
Indeed, the Debian Security Team is very proactive on issues like this. :smile:
